content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,502 @@
+---
+title: "COVID-19 RDT Reference Application"
+linkTitle: "COVID-19 RDT"
+weight: 
+description: >
+ A customizable CHT application with support for 3rd party RDT integration   
+relatedContent: >
+  apps/reference/forms/app/#android-app-launcher
+  apps/reference/forms/app/#cht-xform-widgets
+  apps/reference/forms/app/#parse-timestamp-to-date
+---
+
+Medic has worked with [FIND](https://www.finddx.org) to provide a reference application for rapid diagnostic tests (RDTs) for COVID-19 in the CHT.  By leveraging the expertise of third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), you can customize this application in your current or future deployment of the CHT. 
+
+You can find the code for this application in Medic's main [CHT Core repository on GitHub](https://github.com/medic/cht-core/tree/master/config/covid-19).
+
+## Problem Being Addressed
+
+The original call for proposals best describes why Medic created this app: 
+
+> FIND is working towards supporting countries in implementing an effective test-trace-isolate response using digital tools. To this end, FIND is looking to partner with leading digital solution providers to accelerate the development and deployment of a set of minimum functionalities for the collection of COVID RDT data and supporting incorporation of these functionalities into existing digital tools for use in low- and middle-income country settings.
+
+## Solution Overview
+
+This reference app provides a base layer of functionality that you can easily customize to meet the needs of your health program.  Specifically, this application has examples of:
+
+ * Provisioning COVID-19 RDT tests 
+ * Capturing COVID-19 RDT results
+ * Storing all RDT data, including pictures, in the CHT
+ * Best practices of Health Facility, CHW, patient and RDT hierarchy in the CHT
+ 
+ Your instance of CHT needs to be on v3.13+ which has features developed to support this reference application, including:
+ 
+  * **Display Base64 Image** - show ASCII encoded images inline
+  * **Android App Launcher** - integrate with third party apps like the RD-Toolkit
+  * **Parse Timestamp to Date** -  convert epoch time stamps (`1628945040308`) to your desired format (`Sun Mar 19 13:25:08`).
+  
+  For more information on these features, see the ["Related Content"](#related-content).
+
+## Workflow
+
+There are three main components to this application:
+ * The CHT forms to provision and capture RDTs (Green)
+ * Using a third-party app to use an RDT (Yellow)
+ * Tasks to remind CHWs to complete an RDT that has been started (Blue) 
+
+{{< figure src="flow.png" link="flow.png" class="right col-12 col-lg-12" >}}
```

Comment:
The diagram uses some notations differently than we do in other diagrams (eg blocks for _parts of_ a form, rather than individual forms). Let's chat directly about some changes that could simplify it.

Reply:
for sure! I knew that i was deviating the well trod-path, but thought it was worth it to denote the different apps, tasks and forms involved.